src/content-helper/dashboard-page/pages/traffic-boost/store.ts

```
@@ -0,0 +1,735 @@
+				return {
+					...state,
+					inboundLinksTab: {
+						...state.inboundLinksTab,
+						links: [ action.link, ...state.inboundLinksTab.links ],
+					},
+					selectedLink: action.select ? action.link : state.selectedLink,
+				};
+			case 'REMOVE_INBOUND_LINK':
+				const remainingLinks = state.inboundLinksTab.links.filter(
+					( link ) => link.uid !== action.link.uid
+				);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)